src/Aspire.Hosting.Docker/DockerComposePublishingContext.cs

```diff
@@ -258,4 +260,83 @@ private static void HandleComposeFileVolumes(DockerComposeServiceResource servic
             composeFile.AddVolume(newVolume);
         }
     }
+
+    private void HandleComposeFileBindMounts(DockerComposeServiceResource serviceResource)
+    {
+        foreach (var volume in serviceResource.Volumes.Where(volume => volume.Type == "bind"))
+        {
+            if (string.IsNullOrEmpty(volume.Source))
+            {
+                continue;
+            }
+
+            try
+            {
+                // Determine the destination path for copying the bind mount source
+                var serviceDir = Path.Combine(OutputPath, serviceResource.Name);
+                Directory.CreateDirectory(serviceDir);
+
+                string copiedSourcePath;
+                if (File.Exists(volume.Source))
+                {
+                    // Handle file bind mount
+                    var fileName = Path.GetFileName(volume.Source);
+                    copiedSourcePath = Path.Combine(serviceDir, fileName);
+                    File.Copy(volume.Source, copiedSourcePath, overwrite: true);
```

Comment:
We should be careful to avoid name collisions if multiple files with the same name are being bound into a container. It might make sense to put each copied file or folder into a directory that's a hash of `volume.Source` to avoid the issue. We should do the same in `HandleComposeFileConfig` which also copies files to the output folder.